react/testUtils/test-utils.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import * as reactTestingLibrary from 'react-testing-library'
+import { IntlProvider } from 'react-intl'
+import defaultStrings from '../../messages/en-US.json'
+
+const customRender = (node, options) => {
+  const rendered = reactTestingLibrary.render(
+    <IntlProvider messages={defaultStrings} locale="en-US">
+      {node}
+    </IntlProvider>,
+    options
+  )
+
+  return {
+    ...rendered,
+    rerender: newUi =>
+      customRender(newUi, {
+        container: rendered.container,
+        baseElement: rendered.baseElement,
+      }),
+  }
+}
+
+// re-export everything
+module.exports = {
+  ...reactTestingLibrary,
+  render: customRender,
```

Comment:
Why not call this new render something like `renderWithIntl` and just use it when the Component uses Intl?

Reply:
To keep the API the same as react-testing-library

Reply:
Related docs: https://testing-library.com/docs/react-testing-library/setup#custom-render